### PR TITLE
Conference UI update

### PR DESCRIPTION
**Pull request by lpil**
trippy-hallway

http://staging.react.london/conference

### CODE REVIEW

**Comment by AndrewBestbier** [comment]
Navbar tabs are supposed to be 50px from designs, here they are 55px

File: shared/components/NavigationBar/index.scss [hunk 1]
```diff
@@ -1,59 +1,76 @@
...
+    padding: 0;
 
     a {
+      $border-bottom-size: 3px;
+
       text-decoration: none;
-      text-transform: uppercase;
       font-weight: bold;
-      font-size: 14px;
+      font-size: 20px;
```
Comment (AndrewBestbier) [inline shared/components/NavigationBar/index.scss]: 18px

**Comment by lpil** [comment]
How do you get these numbers?

**Comment by AndrewBestbier** [comment]
Sketch :)

**Comment by AndrewBestbier** [comment]
Badger is behind banner in Safari

**Comment by AndrewBestbier** [comment]
Is the share your ideas supposed to link to our slack channel?

**Pull request merged**